src/Features/Core/Portable/Diagnostics/DefaultDiagnosticModeServiceFactory.cs

```diff
@@ -5,32 +5,31 @@
 using System;
 using System.Collections.Generic;
 using System.Composition;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Experiments;
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Options;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.Diagnostics
+namespace Microsoft.CodeAnalysis.Diagnostics
 {
-    [ExportWorkspaceServiceFactory(typeof(IDiagnosticModeService), ServiceLayer.Host), Shared]
-    internal class VisualStudioDiagnosticModeServiceFactory : IWorkspaceServiceFactory
+    [ExportWorkspaceServiceFactory(typeof(IDiagnosticModeService), ServiceLayer.Default), Shared]
+    internal class DefaultDiagnosticModeServiceFactory : IWorkspaceServiceFactory
```

Comment:
there didn't seem to be really anything in the vs service that was VS specific, so moving down and making default to allow me to test these code paths.